frontend/src/libs/components/button/button.tsx

```diff
@@ -8,15 +8,21 @@ import styles from './styles.module.scss';
 type Properties = {
   label: string;
   type?: 'button' | 'submit';
-  iconName?: IconName;
-  iconColor?: ValueOf<typeof IconColor>;
+  iconName?: IconName | undefined;
```

Comment:
btw do we need this `| undefined`?

Reply:
@Liza-Veis yep, because in card iconColor is optional